android/app/src/main/python/mmrelay/meshtastic_utils.py

```
@@ -0,0 +1,1051 @@
+        if config and "matrix_rooms" in config:
+            matrix_rooms = config["matrix_rooms"]
+
+    with meshtastic_lock:
+        if meshtastic_client and not force_connect:
+            return meshtastic_client
+
+        # Close previous connection if exists
+        if meshtastic_client:
+            try:
+                meshtastic_client.close()
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)